specs/proposal/542678_improve_bundle_image_provided_shape/improve_bundle_image_provided_shape.textile

```diff
@@ -41,7 +41,7 @@ This feature uses the existing API and does not need new API or API changes.
 
 h3. User Interface Changes
 
-The extension point tags _colorAttribute_, _borderColorAttribute_ and __borderSizeAttribute__ will now provide a sub attribute (not mandatory).
+The extension point tags @colorAttribute@, @borderColorAttribute@ and @borderSizeAttribute@ will now provide a sub attribute (not mandatory).
```

Comment:
Maybe the format of the extension point will be different.
`<image imagePath="myPath" label="myLabel">
        <backgroundcolor colorIdentifier="stop8224-2" colorAttribute="style"/>
...
`

instead of 

`<image
           imagePath="/org.eclipse.sirius.tests.swtbot/images/donut.svg"
           label="Donut">
        <colorIdentifier
              colorIdentifier="stop8224-2">
        </colorIdentifier>
        <colorAttribute
              colorAttribute="style">
        </colorAttribute>...`

Reply:
Limitation identified in "Implementation choices and tradeoffs"